### DISCUSSION TITLE
NPM运行无法启动

### DISCUSSION THREAD

**Discussion by ShinramenSashimi** [1715173329/UnblockNeteaseMusic#679] [Q&A] [answered]
如图
出现Uncaught SyntaxError: Invalid or unexpected token
![image](https://user-images.githubusercontent.com/105289911/167625907-218b3c2b-c0b5-406d-a25f-b6d816b8a36a.png)

**Comment by pan93412** [comment, marked as answer]
在 `cmd`（命令提示元）點開，不是在 Node.js 終端打開。

**Reply by ShinramenSashimi** [reply]
![image](https://user-images.githubusercontent.com/105289911/167795994-e6b698e1-1867-44ef-97b6-0cfb206129f2.png)
用CMD运行出现了这个问题

**Reply by pan93412** [reply]
h! @1715173329

**Reply by 1715173329** [reply]
我没有 Windows 下的 nodejs 环境，如果 npx 无法使用的话，可以克隆仓库直接执行 app.js。

**Reply by ShinramenSashimi** [reply]
> 我没有 Windows 下的 nodejs 环境，如果 npx 无法使用的话，可以克隆仓库直接执行 app.js。

直接运行app.js也会出现这样的问题
![image](https://user-images.githubusercontent.com/105289911/167814571-53be2e3a-7233-42b1-b472-2d084ef641fc.png)

**Reply by 1715173329** [reply]
az，你这根本不是 nodejs 在解析这个文件啊，被微软 vbs 解析器接管了。。
需要 cmd 执行 node app.js
